### PR TITLE
Add required steps for installing OSS Calico on Openshift v4.16+ on AWS

### PR DESCRIPTION
**Pull request by coutinhop**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico/getting-started/kubernetes/openshift/installation.mdx [hunk 1]
```diff
@@ -68,6 +68,38 @@ Now generate the Kubernetes manifests using your configuration file:
 openshift-install create manifests
 ```
 
+:::note
+
+If you are using OpenShift **v4.16 or newer** on **AWS**, at this point you need to edit the OpenShift cluster-api manifests
```
Comment (radTuti) [inline calico/getting-started/kubernetes/openshift/installation.mdx]:
```suggestion
For OpenShift **v4.16 or newer** on **AWS**, configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic by editing the OpenShift cluster-api manifests
```

File: calico_versioned_docs/version-3.28/getting-started/kubernetes/openshift/installation.mdx [hunk 2]
```diff
@@ -68,6 +68,38 @@ Now generate the Kubernetes manifests using your configuration file:
 openshift-install create manifests
 ```
 
+:::note
+
+If you are using OpenShift **v4.16 or newer** on **AWS**, at this point you need to edit the OpenShift cluster-api manifests
+that configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic in your cluster.
```
Comment (radTuti) [inline calico_versioned_docs/version-3.28/getting-started/kubernetes/openshift/installation.mdx]:
```suggestion
For OpenShift **v4.16 or newer** on **AWS**, configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic by editing the OpenShift cluster-api manifests
```

File: calico_versioned_docs/version-3.29/getting-started/kubernetes/openshift/installation.mdx [hunk 3]
```diff
@@ -68,6 +68,38 @@ Now generate the Kubernetes manifests using your configuration file:
 openshift-install create manifests
 ```
 
+:::note
+
+If you are using OpenShift **v4.16 or newer** on **AWS**, at this point you need to edit the OpenShift cluster-api manifests
+that configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic in your cluster.
```
Comment (radTuti) [inline calico_versioned_docs/version-3.29/getting-started/kubernetes/openshift/installation.mdx]:
```suggestion
For OpenShift **v4.16 or newer** on **AWS**, configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic by editing the OpenShift cluster-api manifests
```

File: calico/getting-started/kubernetes/openshift/installation.mdx [hunk 4]
```diff
@@ -68,6 +68,38 @@ Now generate the Kubernetes manifests using your configuration file:
...
+        fromPort: 5473
+        protocol: tcp
+        toPort: 5473
+```
+
+Alternatively, you can use the [yq tool](https://github.com/kislyuk/yq) to patch the file with a single command:
+
+```
+yq -y -i '.spec.network.cni.cniIngressRules += [{ "description": "BGP (calico)", "fromPort": 179, "protocol": "tcp", "toPort": 179 }, { "description": "IP-in-IP (calico)", "fromPort": -1, "protocol": "4", "toPort": -1 }, { "description": "Typha (calico)", "fromPort": 5473, "protocol": "tcp", "toPort": 5473 }]' cluster-api/02_infra-cluster.yaml
+```
```
Comment (radTuti) [inline calico/getting-started/kubernetes/openshift/installation.mdx]: `yq` commands tends to be heavly dependant on the vers so perharps we do not want to include `yq` here
```suggestion
```

Comment (ctauchen) [reply calico/getting-started/kubernetes/openshift/installation.mdx]: I agree. Better to present one way to complete a task.

File: calico_versioned_docs/version-3.28/getting-started/kubernetes/openshift/installation.mdx (hunk 2 above)
Comment (ctauchen) [reply calico_versioned_docs/version-3.28/getting-started/kubernetes/openshift/installation.mdx]: I also prefer one sentence per line. Much easier to review in git.

**Review by ctauchen — COMMENTED** [review]
LGTM, with suggestions.

@coutinhop This PR covers OSS 3.28+. Am I right to assume this does not apply to 3.27? What about CE?

**Comment by coutinhop** [comment]
> LGTM, with suggestions.
> 
> @coutinhop This PR covers OSS 3.28+. Am I right to assume this does not apply to 3.27? What about CE?

Thanks @ctauchen and @radTuti ! I've addressed your comments. Yes, this doesn't apply to v3.27 as the product changes to support OCP v4.16+ were only backported to v3.28+. The changes for Calico EE are still in flight, I'll change the docs once the product changes are merged.

**Review by radTuti — APPROVED** [review]

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**